.github/workflows/ci.yml

```diff
@@ -37,3 +37,24 @@ jobs:
           platforms: amd64,arm64,ppc64le
       - name: Build and Test Docker Image
         run: docker/build.sh
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.12.1
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+          check-latest: true
+      - name: Lint chart
+        run: helm lint --strict helm/trino-gateway
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.10.0
+        with:
+          cluster_name: tgw
+      - name: load images
+        run: kind load docker-image --name tgw trino-gateway:10-SNAPSHOT-amd64 && kind load docker-image --name tgw trino-gateway:10-SNAPSHOT-arm64
+      - name: Run chart-testing (install)
+        run: helm install tgw helm/trino-gateway  --set "image.repository=trino-gateway" --set "image.tag=$(cat ./trino-gateway-version.txt | tr -d '\n')"  --set "image.tag=10-SNAPSHOT-amd64" --set "limits.memory=100Mi" --set "limits.cpu=0"
+      - name: wait for deployment
+        run: kubectl rollout status deployment tgw-trino-gateway
```

Comment:
tgw-trino-gateway .. maybe just tgw or trino-gateway or something simple

Reply:
yeah .. I think we should go with test or so instead and also clarify where that comes from